#Camera topic here
camera_topic: "/usb_cam/image_raw"
queue_size: 1

# Set your detection topic here
detection_topic: "/darknet_ros/bounding_boxes"

# Set your output topic here
tracker_topic: "/sort_track"

# Threshold for Costs between detections and tracks
cost_threhold: 15

# min_hits
min_hits: 5

visualize_raw_detections: false
visualize_filtered_detections: false
visualize_tracked_objects: true
visualize_unreliable_objects: false


non_max_suppression_th: 0.6


# max age
max_age: 15
IOU_th_association: 0.5 #For tracking Needs to be fairly low to provide robustness to difference in bounding box size
VerifyByDetectionRate: true

image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true
